### ISSUE TITLE
zopen newbie nits

### ISSUE THREAD

**Issue by covener** [ZOSOpenTools/meta#248]
I recently went into my simple system to do some upgrades, I had a few key packages and meta from January.
Some quick feedback

1. I had absolutely no idea how to upgrade the boot directory.  I am not sure it was possible with the older meta I had, but I think it should be prominent on the users website (zopen upgrade -d . from within boot)
2. Consider whether `zopen install` should print the usage instead of slowly listing off the available ports.
3. the usage printed with -h doesn't show the package name optional argument

    ```
    Syntax: zopen install [<option>]*
    ```

    should be e.g

    ```
    Syntax: zopen install [<option>]* [package-name]
   ```

4. extra parameters are ignored, I naively did `zopen install bash vim` expecting two packages to be installed
5. unknown flags, like -X are treated as package names rather than triggering usage
6. `zopen upgrade -h` gives you `zopen install -h` output
7. no apparent way to `zopen --version` to run before you go off to post on slack :)

**Comment by MikeFultonDev** [comment]
Great feedback. Thank you. @IgorTodorovskiIBM @HarithaIBM fyi

**Comment by IgorTodorovskiIBM** [comment]
+1, great feedback. @DevonianTeuchter, curious if any of these are addressed in your fork?

**Comment by MikeFultonDev** [comment]
All of these are fixed now except #6 and #7. #7 has another issue from Lionel. I will open one up for #6
